releases/cockroachcloud-12092020.md

```diff
@@ -0,0 +1,28 @@
+---
+title: What&#39;s New in CockroachCloud
+toc: true
+summary: Additions and changes in CockroachCloud.
+---
+
+## December 9, 2020
+
+Get future release notes emailed to you:
+
+<div class="hubspot-install-form install-form-1 clearfix">
+    <script>
+        hbspt.forms.create({
+            css: '',
+            cssClass: 'install-form',
+            portalId: '1753393',
+            formId: '39686297-81d2-45e7-a73f-55a596a8d5ff',
+            formInstanceId: 1,
+            target: '.install-form-1'
+        });
+    </script>
+</div>
+
+### General changes
+
+New CockroachCloud clusters will now run CockroachDB [v20.2.2](v20.2.2.html).
+
+- CockroachCloud is now offering [larger machine sizes](../cockroachcloud/create-your-cluster.html#step-2-select-the-cloud-provider) to be configured directly in the Console. You will now be able to select from four options in the create cluster workflow. The pricing has also been updated for newly created clusters. Existing clusters are not impacted by the pricing changes.
```

Comment:
Might make sense to link to the new pricing here to save me a search.  Since the main thing this note made me wonder was: how much more is it now?